app/models/results_submission.rb

```diff
@@ -12,7 +12,7 @@ class ResultsSubmission
   validates :schedule_url, presence: true, url: true
 
   validate do
-    if results_validator.has_errors?
+    if results_validator.errors?
```

Comment:
Same as `results?` above. Perhaps `any_errors?`

Reply:
erorred? I think is good here